extended_mypy_django_plugin/_plugin/annotation_resolver.py

```diff
@@ -277,8 +285,10 @@ def _instances_from_aliases(
 
             try:
                 yield from self.lookup_alias(alias)
-            except AssertionError:
-                self.fail(f"Failed to create concrete alias instance for '{model}' ({alias})")
+            except FailedLookup as error:
+                self.fail(
+                    f"Failed to create concrete alias instance for '{model}' ({error}) (this is likely a bug in extended_mypy_django_plugin)"
+                )
```

Comment:
It seems like we have two different types of error here. Perhaps we should catch different types for them, and give specific errors for each one?

Reply:
given it's likely this only happens when there is a bug in the plugin, I'm unsure that would add value. It would kinda just shuffle the message around a bit but the result would largely be the same. (I'm substituting in here the specifics via `({error})` but the reset of the message is still relevant to both errors.

Reply:
I'll merge this for now, but please bring it up again if you still think I should change it :)